data/pools/transactionPool_test.go

```diff
@@ -1099,6 +1102,115 @@ func BenchmarkTransactionPoolPending(b *testing.B) {
 	}
 }
 
+func BenchmarkTransactionPoolRecompute(b *testing.B) {
+	b.Log("Running with b.N", b.N)
+	poolSize := 100000
+	numOfAccounts := 100
+	numTransactions := 75000
+	blockTxnCount := 25000
+
+	myVersion := protocol.ConsensusVersion("test-large-blocks")
+	myProto := config.Consensus[protocol.ConsensusCurrentVersion]
+	if myProto.MaxTxnBytesPerBlock != 5*1024*1024 {
+		b.FailNow() // intended to use with 5MB blocks
+	}
+	config.Consensus[myVersion] = myProto
+
+	// Generate accounts
+	secrets := make([]*crypto.SignatureSecrets, numOfAccounts)
```

Comment:
oh there was another version of this where I was signing these txns. But for this benchmark it is assumed all the signatures have all been verified before txns get the pool. Will remove. I think it is good to have valid addresses from the ledger though, because I had the recipient using random addresses before, and I think when I switched to using a known destination address from the ledger this led to a lot of DB lookups disappearing from the profile (we aren't caching missing lookups in this case?)